consensus/polybft/contractsapi/gen_sc_data.go

```diff
@@ -1,8 +1,10 @@
 package contractsapi
```

Comment:
Consider adding this file to git ignore as it is result of generation process. Rationale is to prevent merge conflicts around it in the future. WDYT?

Reply:
I thought about it. The idea was to store generated file to be able to compile binary without smart contracts.
Regarding updates - it'll require updating core-contracts branch. If we update it in 2 separated branches(which is strange), we'll have to resolve conflict in core-contracts commit also.

Reply:
I'd prefer to have it commited and update when we update core-contracts branch